### PR TITLE
.travis.yml for integrate travis ci

### PR DESCRIPTION
**Pull request by imlidian**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+script:
+  - git clone --depth=10 --branch=master https://GIT_URL incubator-servicecomb-website
```
Comment (WillemJiang) [inline .travis.yml]: This script doesn't generate the git page for use.

File: .travis.yml [hunk 2]
```diff
@@ -1,28 +1,12 @@
...
+  - cp -r java-chassis-reference incubator-servicecomb-website/users/servicecomb-java-chassis-doc
+	- cd incubator-servicecomb-website
 after_success:
-
-  #- git checkout -b asf-site
-  - git add .
+  - git checkout -b master
+  - git add users/servicecomb-java-chassis-doc/java-chassis-reference/
   - git commit -m "update the website docs source"
   - git push --force https://$DEPLOY_TOKEN@$GIT_URL maser
```
[the remark this answers is not in the document]
Comment (imlidian) [reply .travis.yml]: yes, It has the authorization to do this. As you can see DEPLOY_TOKEN in Travis ci environment variable, so we can check in directly.

**Comment by imlidian** [comment]
We can publish the gitbook anywhere you want. Just set the environment variable in Travis CI.
just like https://huaweicse.github.io/servicecomb-java-chassis-doc/

**Comment by imlidian** [comment]
follow steps to make sure auto push work:
Github Token Generation
https://github.com/settings/developers
step1: logo in and into Personal access tokens
step2: New personal access token
-input Token description
-select Select scopes
-click Generate Token
-copy token (ex:f59d338924089b1bc0e1ea0b6fb8e8f1ddxxxxxx)
step3: make sure save to your token copy!
GitHub ref link: https://files.gitter.im/ServiceCombUsers/Lobby/8OvN/image.png

Travis-ci Intergration 
https://travis-ci.org/microDevOps/ServiceComb.github.io
step1 login and intergrate Travis-ci
step2 select more option --> setting
step3 input Environment Variables
- DEPLOY_TOKEN:f59d338924089b1bc0e1ea0b6fb8e8f1ddxxxxxx
- PUSH_TARGET_URL: github.com/apache/incubator-servicecomb-docs
( or github.com/staging_environment/incubator-servicecomb-docs)
step4 click restart button or commit your new update from github 
Trais CI ref link:https://files.gitter.im/ServiceCombUsers/Lobby/8Ncg/image.png

**Pull request merged**